components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/system-calls-cairo1.adoc

```diff
@@ -177,7 +177,7 @@ Deploys a new instance of a previously declared class.
 `_class_hash_`:: The class hash of the contract to be deployed.
 `_contract_address_salt_`:: The salt, an arbitrary value provided by the sender, used in the computation of the xref:Smart_Contracts/contract-address.adoc[contract's address].
 `_calldata_`:: The constructor's calldata. An array of felts.
-`_deploy_from_zero_`:: A flag used for the contract address computation. If not set, the caller address will be used as the new contract's deployer address, otherwise 0 is used.
+`_deploy_from_zero_`:: A flag that determines whether the deployer’s address affects the computation of the contract address. When not set, or when set to `FALSE`, the caller address is used as the new contract's deployer address. When set to `TRUE`, 0 is used.
```

Comment:
But then why is it always ZERO in the libs I looked at?
When are we using `FALSE` to use the caller address?

Reply:
Maybe you haven't seen it because it's also allowed to just _not_ set it.
@arielelp?

Reply:
See [here](https://github.com/OpenZeppelin/cairo-contracts/blob/9f34e3c52299af342fd82499805887361bc2fd1b/src/presets/universal_deployer.cairo#L52)